app/jobs/send_fill_invitation/intervention_job.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class SendFillInvitation::InterventionJob < ApplicationJob
+  def perform(intervention_id, existing_users_emails, non_existing_users_emails, health_clinic_id = nil)
+    intervention = Intervention.find(intervention_id)
+    health_clinic = HealthClinic.find_by(id: health_clinic_id)
+
+    existing_users_emails.each do |email|
+      user = User.find_by(email: email)
+      next if user && !user.email_notification
```

Comment:
The same as for `SendFillInvitation::SessionJob`

Reply:
The same as for `SendFillInvitation::SessionJob`